azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/ExternalChildResource.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.azure.management.compute;
```

Comment:
Ok.. its used only in compute, can be moved to resource once we identify similar pattern in other packages.